components/items/ItemsGrid/useItemsGrid.ts

```diff
@@ -127,6 +127,25 @@ export function useFetchSearch({
       orderBy: getRouteQueryOrDefault(route.query.sort, ['blockNumber_DESC']),
     }
 
+    type SortBy = string | null
```

Comment:
Small stuff. I would move this type of definition outside this function.

Reply:
prefer inline it?
 it's sole purpose is to make to fn signature tiny bit easier to read

Reply:
No, I do not mean to inline it. I prefer to move the type definition `type SortBy = string | null` to the top of this file ( Line 23). It would be more clear to separate the type and function.

It doesn’t matter, inline is also possible